bootstrap/bootstrap.ps1

```diff
@@ -45,6 +45,11 @@ Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/script
 $rootCliVersion = Join-Path $RepositoryRoot ".cliversion"
 $dotNetCliVersion = Get-Content $rootCliVersion
 
+if (-Not (Test-Path $CliLocalPath))
```

Comment:
We should probably have this repo use the local script instead of pulling from the net. Thoughts?